### DISCUSSION TITLE
missing library after updating to angular 13

### DISCUSSION THREAD

**Discussion by abdallahkadour** [NativeScript/NativeScript#9859] [General]
After upgrading to angular 13 I got the following problem which leads to build fail:

 ```A problem occurred configuring project ':app'.
Could not find com.bm.photoview:library:1.4.1.
Required by:
    project :app
```
Does anyone has the same problem? Thanks in advance.

**Comment by rigor789** [comment]
This is due to the update to `@nativescript/android` - we're no longer using jcenter or bintray by default as those have been deprecated repositories for a while and can stop working at any time now...

https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

The plugin would need to be updated - but in the meantime you can try adding: 
```groovy
allprojects {
    repositories {
      jcenter()
    }
}
```

to your `App_Resources/Android/.../app.gradle` and then `ns clean` and rebuild/re-run.

**Comment by abdallahkadour** [comment]
@rigor789 Thanks for your reply, but this didn't solve the issue. I excluded the library as follows:
```
dependencies {
  //Execute problematic library
  configurations.all {
        exclude group: 'com.bm.photoview', module: 'library'
    }
}
```

**Comment by nikoTM** [comment]
To add more context to this in case it helps someone else, for me the dependency was coming from `nativescript-mediafilepicker` here https://github.com/jibon57/nativescript-mediafilepicker/blob/master/src/platforms/android/include.gradle#L16  so excluding it might cause issues if you are using something that depends on it.
